### DISCUSSION TITLE
Possibility to add On Click Action to Container?

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#3549] [Ideas]
Hello,

As I know, currently only Button and Icon can trigger On Click Action, it would be really helpful if the container can has the On Click Action, then we can easier trigger an action almost anywhere without a button or icon, it's really helpful especially on the mobile view.

**Comment by mjashanks** [comment]
Thanks - this is a duplicate of https://github.com/Budibase/budibase/discussions/2256 - please upvote that one :)
